README.md

```diff
@@ -1,2 +1,81 @@
 # apostrophe-svg-sprites
-SVG sprite elements as pieces
+
+`apostrophe-images` for SVGs in external sprite maps!
+![Managing sprites with apostrophe-svg-sprites](https://github.com/apostrophecms/apostrophe-svg-sprites/raw/1.0.0/demo.gif)
+
+This bundle provides a piece subclass that manages and renders SVG sprites referenced from external maps and an accompanying widget for display.
+
+This bundle assumes you are compiling/managing the SVG maps on your own (Gulp, Webpack, manual) and leaving them in a publicly accessible place (`/public` for example).
+
+This module is *not* an interface for uploading SVG files or pasting SVG markup.
+
+Bundle consists of 
+
+* `apostrophe-svg-sprites` as a piece subclass for managing SVGs, similar to `apostrophe-images`
+* `apostrophe-svg-sprites-widgets` as a way to render the SVG on a page
+
+`apostrophe-svg-sprites` supports multiple source maps for multiple sets of SVGs.
+
+## Example configuration
+
+```javascript
+// in app.js
+// We must declare the bundle!
+bundles: [ 'apostrophe-svg-sprites' ],
+modules: {
+  'apostrophe-svg-sprites': {
+    maps: [
+      {
+        label: 'Social Media Icons',
+        name: 'social',
+        file: 'svg/social.svg'
+      },
+      {
+        label: 'Places Icons',
+        name: 'places',
+        file: 'svg/places.svg'
+      }
+    ]
+  },
+  'apostrophe-svg-sprites-widgets': {},
```

Comment:
Correct, although you don't have to declare bundles you *do* have to turn on modules if you want them.

Reply:
(Improve modules are an exception because turning them on is very counterintuitive and they generally ship with an "on switch" flag of their own if they are something you might not always want for every content type, like import.)